execution/executor-benchmark/src/main.rs

```diff
@@ -343,11 +360,32 @@ fn main() {
     AptosVM::set_num_shards_once(opt.pipeline_opt.num_executor_shards);
     NativeExecutor::set_concurrency_level_once(opt.concurrency_level());
 
+    let config = ProfilerConfig::new_with_defaults();
+    let handler = ProfilerHandler::new(config);
+    let mut cpu_profiler = handler.get_cpu_profiler();
+    let mut memory_profiler = handler.get_mem_profiler();
+    let memory_profiling = opt.profiler_opt.memory_profiling;
+    let cpu_profiling = opt.profiler_opt.cpu_profiling;
+
+    if cpu_profiling {
+        let _cpu_prof = cpu_profiler.start_profiling();
```

Comment:
move `let mut cpu_profiler = handler.get_cpu_profiler();` here? similarly for `memory_profiler`

Reply:
had to move it back outside the if (my understanding of your comment was putting it inside) because end_profiling also uses the cpu_profiler object and caused lint errors

Reply:
ok, make sense